### PR TITLE
DEM/mask bug-fixes/enhancements. Duplicate products now removed

### PR DESCRIPTION
**Pull request by sssangha**
Bug-fixes/enchancements:
1. Fixed bug in prep_dem/prep_mask which may lead to cropping inconsistent with interferometric grid when user provides custon DEM/mask.
2. In continuous_time, search for duplicate standard products, and then print warning and delete duplicate products from list.
3. Clean up sections where %overlap is computed for interferometric grid vs user bounding box in merged_productbbox

### CODE REVIEW

File: tools/ARIAtools/extractProduct.py [hunk 1]
```diff
@@ -596,11 +591,7 @@ def tropo_correction(full_product_dict, tropo_products, bbox_file, prods_TOTbbox
         bbox=Polygon(np.column_stack((np.array([bbox[2],bbox[3],bbox[3],bbox[2],bbox[2]]),
                             np.array([bbox[0],bbox[0],bbox[1],bbox[1],bbox[0]]))))
         save_shapefile(i+'.shp', bbox, 'GeoJSON')
-        bbox_area=open_shapefile(i+'.shp', 0, 0)
-        bbox_area=user_bbox.intersection(bbox_area)
-        bbox_area=bbox_area.bounds
-        bbox_area=(max(bbox_area[0],bbox_area[2]) - min(bbox_area[0],bbox_area[2]))*(max(bbox_area[1],bbox_area[3]) - min(bbox_area[1],bbox_area[3]))
-        per_overlap=(prods_bbox_area/bbox_area)*100
+        per_overlap=((user_bbox.intersection(open_shapefile(i+'.shp', 0, 0)).area)/(user_bbox.area))*100
```
Comment (dbekaert) [inline tools/ARIAtools/extractProduct.py:594]: Should the message below not be related to overlap warning for tropo?
Its the same message as for a regular GUNW product, so might be confusing.

Comment (sssangha) [reply tools/ARIAtools/extractProduct.py:594]: Yeah, true. How about something like this: "print("WARNING: Common track extent only has %d%% overlap with tropospheric product %s"%(per_overlap, i)+'\n')"

Where "i" points to the tropospheric product referenced in the loop?

Comment (dbekaert) [reply tools/ARIAtools/extractProduct.py:594]: yes that works.

Comment (sssangha) [reply tools/ARIAtools/extractProduct.py:594]: Ok, committed.

**Review by dbekaert — APPROVED** [review]

**Pull request merged**